### PR TITLE
DDT-1058: Remove shipment_event.document_reference

### PR DESCRIPTION
**Pull request by mp-assco**

### CODE REVIEW

File: datamodel/initdb.d/05_reference_implementation_support.sql [hunk 1]
```diff
@@ -687,4 +687,7 @@ CREATE TABLE dcsa_im_v3_0.ebl_solution_provider_type (
 --- DDT-948
 ALTER TABLE dcsa_im_v3_0.equipment_event ADD utilized_transport_equipment_id uuid NULL REFERENCES dcsa_im_v3_0.utilized_transport_equipment(id);
 
+-- DDT-1058
+ALTER TABLE dcsa_im_v3_0.shipment_event ADD document_reference varchar(100);
```
Comment (HenrikHL) [inline datamodel/initdb.d/05_reference_implementation_support.sql:691]: Move up before the UNION commented above...

Comment (mp-assco) [reply datamodel/initdb.d/05_reference_implementation_support.sql]: Also done

File: datamodel/initdb.d/05_reference_implementation_support.sql [hunk 2]
```diff
@@ -53,7 +53,7 @@ UNION ALL
     NULL::text AS equipment_reference,
     NULL::text AS empty_indicator_code,
     shipment_event.document_id AS document_id,
-    shipment_event.document_reference AS document_reference,
+    null::text AS document_reference,
```
[the remark this answers is not in the document]
Comment (HenrikHL) [reply datamodel/initdb.d/05_reference_implementation_support.sql:60]: You have deleted `shipment_event.document_reference AS document_reference` and inserted null::text AS document_reference,` instead - I think this is wrong

[the remark this answers is not in the document]
Comment (HenrikHL) [reply datamodel/initdb.d/05_reference_implementation_support.sql:60]: Please undo your change

File: datamodel/initdb.d/05_reference_implementation_support.sql [hunk 3]
```diff
@@ -4,6 +4,10 @@
 
 -- Implementation specific SQL for the reference implementation.
 BEGIN;
+
+-- DDT-1058
+ALTER TABLE dcsa_im_v3_0.shipment_event ADD document_reference varchar(100);
```
Comment (HenrikHL) [inline datamodel/initdb.d/05_reference_implementation_support.sql:9]:
```suggestion
ALTER TABLE dcsa_im_v3_0.shipment_event ADD document_reference varchar(100) NOT NULL;

```

File: datamodel/initdb.d/05_reference_implementation_support.sql (hunk 2 above)
[the remark this answers is not in the document]
Comment (mp-assco) [reply datamodel/initdb.d/05_reference_implementation_support.sql]: Undone

**Review by HenrikHL — APPROVED** [review]

**Pull request merged**